_assets/css/_landing.scss

```diff
@@ -1197,52 +1122,49 @@ Stylesheet: Landing Page Stylesheet
 .news-slider {
   @include breakpoint(medium down) {
     padding: rem-calc(0 0 1);
-    margin-bottom: calc(#{5.6652083333333vw * 5} + #{rem-calc(96)}) !important;
+    margin-bottom: rem-calc(40) !important;
 
     .grid-container {
       padding: rem-calc(0 0 1);
-      margin-bottom: calc(
-        #{-5.6652083333333vw * 5} + #{rem-calc(-96)}
-      ) !important;
+      margin-bottom: rem-calc(-40) !important;
+    }
+
+    .slick-list {
+      position: relative;
+      z-index: 2;
+    }
 
-      .slick-list {
-        position: relative;
-        z-index: 2;
+    .slick-track {
+      display: flex;
+      align-items: stretch;
+    }
+
+    .slick-slide {
+      display: flex;
+      flex-flow: column nowrap;
+      justify-content: stretch;
+      height: auto;
+      padding: rem-calc(20 28) !important;
```

Comment:
Is the `!important` definitely required?

Reply:
yeah - this overrides the .cell padding applied by foundation.